package.json

```diff
@@ -1,18 +1,20 @@
 {
   "name": "pipeline-graph-view-plugin",
-  "version": "1.0.0",
+  "version": "0.0.0-development",
   "description": "Pipeline Graph visualization for Jenkins pipelines",
   "private": true,
   "type": "module",
   "scripts": {
     "mvnbuild": "npm run build",
-    "mvntest": "npm test",
+    "mvntest": "echo \"Tests should be executed through 'npm test' instead\"",
```

Comment:
we still want this to be run through the maven build so that a full build is tested for both java and javascript.

People can skip the running of the tests easily with e.g. `quick-build` profile and on CI I don't think it matters much